cocos/scripting/js-bindings/manual/network/jsb_websocket.cpp

```diff
@@ -78,6 +79,9 @@ class JSB_WebSocketDelegate : public WebSocket::Delegate
         js_proxy_t * p = jsb_get_native_proxy(ws);
         if (!p) return;
 
+        if (cocos2d::Director::getInstance() == nullptr || cocos2d::ScriptEngineManager::getInstance() == nullptr)
```

Comment:
ok, I think keep them synced is beneficial, who knows when we are gonna merge them